packages/fxa-content-server/app/scripts/lib/cropper.js

```diff
@@ -110,18 +122,43 @@ Cropper.prototype._setupElements = function(container) {
     $sliderEl.val(this.scale);
     this.onZoomIn();
   });
+};
+
+Cropper.prototype.destroy = function(container) {
```

Comment:
I think we can leave an empty parameter here, otherwise I get a `'container' is declared but its value is never read.`